scripts/player/Ability.cs

```diff
@@ -2,21 +2,27 @@
 
 public partial class Ability : Resource
 {
+	// This is the texture for the icon in the UI.
 	[Export]
 	public Texture2D icon { get; set; }
 
+	// This is the animation the player will play when using the ability.
+	// Should we have this here? Can be refactored.
 	[Export]
 	public Animation playerAnimation { get; set; }
 
```

Comment:
Refactor how?

Reply:
Refactored in either removed or a different implementation